cpp/test/CMakeLists.txt

```diff
@@ -4,9 +4,7 @@ cxx_add_compile_options(MSVC /wd4310)
 # Core unit tests don't build under VS2013 without this.
 cxx_add_compile_options(MSVC /EHa)
 
-if (NOT BOND_SKIP_CORE_TESTS)
-    add_subfolder (compat/core "tests/compat/core")
-endif()
+add_subfolder (compat/core "tests/compat/core")
```

Comment:
BOND_SKIP_COMPAT_TESTS

---
In reply to: [130488475](https://github.com/Microsoft/bond/pull/540#discussion_r130488475) [](ancestors = 130488475)